examples/complete/main.tf

```
@@ -97,7 +97,7 @@ module "network_interface" {
 
 module "virtual_network" {
   source  = "terraform.registry.launch.nttdata.com/module_collection/virtual_network/azurerm"
-  version = "~> 1.0"
+  version = "~> 1.2.0"
```

```suggestion
  version = "~> 1.2"
```

This will set a floor of 1.2 (allowing 1.2.1, 1.3, 1.4). Otherwise, only 1.2.x is valid.

---

this has been resolved